src/main/sensors/sensors.h

```diff
@@ -48,6 +48,7 @@ typedef enum {
     SENSOR_MAG = 1 << 3,
     SENSOR_SONAR = 1 << 4,
     SENSOR_GPS = 1 << 5,
+    SENSOR_GPSMAG = 1 << 6,
```

Comment:
I don't think it is. But it is harmless and means there are fewer diffs with betaflight.